fsw/mcp750-vxworks/src/cfe_psp_memory.c

```diff
@@ -319,9 +319,7 @@ int32 CFE_PSP_GetVolatileDiskMem(cpuaddr *PtrToVolDisk, uint32 *SizeOfVolDisk)
 */
 int32 CFE_PSP_InitProcessorReservedMemory(uint32 RestartType)
 {
-    int32 return_code;
-
-    if (RestartType != CFE_PSP_RST_TYPE_PROCESSOR)
+    if (RestartType == CFE_PSP_RST_TYPE_POWERON)
```

Comment:
Hi @thnkslprpt, what was the rationale for changing this to check for equaling POWERON?